clients/Dockerfile

```diff
@@ -9,7 +9,8 @@ RUN yum upgrade -y && \
     yum clean all && \
     rm -rf /var/cache/yum
 RUN yum -y install https://repo.ius.io/ius-release-el7.rpm && \
-    yum install -y python36u-pip voms-clients-java gfal2-all gfal2-util gfal2-python3 xrootd-client\
+    yum install -y python36u-pip voms-clients-java gfal2-all gfal2-util gfal2-python3 xrootd-client \
```

Comment:
@bari12 are these the client images used by everyone? because then i think we should keep out the openssh additions here until everything is done? whaddyathink?